odinbot/agent.py

```
@@ -235,6 +237,12 @@ async def on_ready(self) -> None:
         """Handle bot ready event."""
         logger.info(f'Logged in as {self.user}')
         logger.info("Bot is ready!")
+        # Send a startup message to the configured channel
+        channel = self.get_channel(int(self.channel_id))
+        if channel:
+            await channel.send("Hi everyone!")
+        else:
+            logger.warning(f"Could not find channel with ID {self.channel_id} to send startup message.")
```

If it can't find the channel, should it throw an exception instead of only logging a warning? It's unclear why failing to find the channel is ok.